uint32 foot
uint32 forceaid
uint32 max_force
int32 max_position
int32 zero_position
int32 preload_position
int32 max_velocity
int32 nset_acc
float32 max_pot
float32 pid_kp
float32 pid_ki
int32 pid_umax
int32 pid_umin